src/main/frontend/dicts.cljc

```diff
@@ -295,9 +296,8 @@
         :sync-from-local-changes-detected "Refresh detects and processes files modified on your disk and diverged from the actual Logseq page content. Continue?"
 
         :unlink "unlink"
-        :search (if config/publishing?
```

Comment:
Dictionaries should just be data. Having these conditionals here makes linters and future code organization needlessly difficults

Reply:
<img width="169" alt="image" src="https://user-images.githubusercontent.com/72891/160554248-5b1ef3a2-0cb7-4280-95c4-24b64980f8f0.png">
<img width="132" alt="image" src="https://user-images.githubusercontent.com/72891/160554296-c1ea16b7-4afe-4c25-9ef4-ed349d8a9abd.png">

plural form is not handled well yet. Need a solution other than plain dictionaries.

Reply:
That's a fair point. We would need to think harder on pluralization. Pluralization is difficult enough in english that there are libraries just for it. I can't imagine what a general purpose solution is for 12+ languages